Library/ViewModels/PledgeViewModel.swift

```diff
@@ -831,7 +837,13 @@ private func paymentMethodValid(
     return true
   }
 
-  return backedPaymentSourceId != paymentSourceId
+  if backedPaymentSourceId == paymentSourceId, project.personalization.backing?.status == .errored {
```

Comment:
I think we aren't concerned with the first part of this `if` clause, this can probably just be:
```swift
if project.personalization.backing?.status == .errored {
  return true
} else if backedPaymentSourceId != paymentSourceId {
  return true
}
```
Does that seems accurate?